webix/index.d.ts

```diff
@@ -212,7 +212,7 @@ function message(content:any):void;
 function modalbox(text:string, callback:WebixCallback):HTMLElement;
 function once(code:WebixCallback):void;
 function proto(target:any, ...mixins:any[]):any;
-function protoUI(target:any, view:any, mixins:any):any;
+function protoUI(target: any, ...mixins: any[]): any;
```

Comment:
When you look in the webix source file to the call to protoUI for the rangechart widget you can see that their is indeed a problem with the definition file:
see line 41169 for the call to protoUI and line 41400 for the 2nd argument to that call.

But indeed, probably the view argument can be just made nullable. But would it not be more correct to write:
```javascript
    function protoUI(target: any, view: any, ...mixins?: any[]): any;
```
Make the mixins argument optional instead of the view argument ?